h2o-algos/src/main/java/hex/anovaglm/GenerateTransformColumns.java

```diff
@@ -0,0 +1,151 @@
+package hex.anovaglm;
+
+import hex.DataInfo;
+import water.MRTask;
+import water.fvec.Chunk;
+import water.fvec.NewChunk;
+
+import static hex.anovaglm.AnovaGLMModel.AnovaGLMParameters;
+import static hex.anovaglm.AnovaGLMUtils.findComboMatch;
+
+/***
+ * This class will take two predictors and transform them according to rules specified in Wendy Docs
```

Comment:
Again, please use the link to the document instead of 'Wendy Docs'. It would be much easier for future developers to find it.

Reply:
Will do.

Reply:
Removed.

Reply:
I added the tutorial to the JIRA and referred the link in the comments.